### ISSUE TITLE
redis json

### ISSUE THREAD

**Issue by luluhoc** [DevAngsy/nestjs-redis#337]
Can i use `client.json.set` with this library?

**Comment by liaoliaots** [comment]
Nope, but for now you can use **Redis Stack** via [node-redis](https://github.com/redis/node-redis) or [redis-modules-sdk-ts (rely on ioredis)](https://github.com/danitseitlin/redis-modules-sdk-ts).

**Comment by waltersfilho** [comment]
Any prioritization plans for this?
